dask/widgets/templates/array.html.j2

```diff
@@ -1,4 +1,7 @@
 <table>
+    <caption style="white-space: nowrap; overflow: hidden; caption-side: bottom;">
+      Dask graph: {{ array.npartitions }} Chunks in {{ layers }}, Data type: {{ array.dtype }} {{ array._meta | type | typename }}
```

Comment:
"Chunks" should probably not be capitalized here since it is in the middle of a sentence.

Reply:
In that case, should I change "Graph Layer" into "graph layer" in this line as well?

https://github.com/dask/dask/blob/main/dask/array/core.py#L1648

<img width="1087" alt="Screen Shot 2022-09-30 at 14 03 34" src="https://user-images.githubusercontent.com/13074130/193194226-5de24988-e46b-475f-8484-cb13d7efe962.png">

Reply:
I think that's a good idea, yes